Source/API/EbApi.h

```diff
@@ -186,11 +186,11 @@ typedef struct EB_H265_ENC_CONFIGURATION
 
     /* Random access.
      *
-     * 1 = CRA, open GOP.
-     * 2 = IDR, closed GOP.
+     * -1  = CRA, open GOP.
+     * >=0 = IDR, closed GOP, and the value is headers insertion interval.
```

Comment:
If customer deploy already, the value change will break their product.
Is there a way to keep backward compatibility?

Reply:
After a discussion with Li Jing, made such design.
It will not break the current deployment. 1 and 2 are still supported, but with different behaviors. IDR or CRA, the diff is minor.

Reply:
@tianjunwork , I think this will cause the smallest impact compared to others.
Let us high light the change some where, either in release documents or in our doc somewhere